library/src/interactor_impl.cxx

```diff
@@ -259,10 +259,14 @@ class interactor_impl::internals
       const char* commandWithArgs = static_cast<const char*>(data);
       self->Interactor.SetCommandBuffer(commandWithArgs);
     }
-    else if (event == vtkF3DConsoleOutputWindow::ShowEvent ||
-      event == vtkF3DConsoleOutputWindow::HideEvent)
+    else if (event == vtkF3DConsoleOutputWindow::ShowEvent)
```

Comment:
maybe add a comment explaining when this event is received (when cliking on the console badge...)